.github/workflows/unit_tests_with_latest_deps.yml

```diff
@@ -37,33 +38,48 @@ jobs:
       - name: Install woodwork - tests requirements
         run: |
           python -m pip install unpacked_sdist/[test]
-      - if: ${{ matrix.libraries == 'spark' }}
-        name: Install woodwork - spark requirements
+      - name: Install Dask and Spark
         run: |
           sudo apt update
           sudo apt install -y openjdk-11-jre-headless
           JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
           python -m pip install unpacked_sdist/[spark]
-      - if: ${{ matrix.libraries == 'dask' }}
-        name: Install woodwork - dask requirements
-        run: |
           python -m pip install unpacked_sdist/[dask]
-      - if: ${{ matrix.python_version != 3.8 }}
-        name: Run unit tests (no code coverage)
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'Serialization' }}
+        name: Run Serialization Unit Tests (no code coverage)
         run: |
           cd unpacked_sdist
-          pytest woodwork/ -n 2
+          pytest woodwork/tests/accessor/test_serialization.py -n 2 --durations 0
         env:
           PYARROW_IGNORE_TIMEZONE: 1
           ALTERYX_OPEN_SRC_UPDATE_CHECKER: False
-      - if: ${{ matrix.python_version == 3.8 }}
-        name: Run unit tests with code coverage
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'Non-serialization' }}
+        name: Run Non-serialization Unit Tests (no code coverage)
+        run: |
+          cd unpacked_sdist
+          pytest woodwork/ -n 2 --ignore=woodwork/tests/accessor/test_serialization.py --durations 0
+        env:
+          PYARROW_IGNORE_TIMEZONE: 1
```

Comment:
You can set the ENV for the whole workflow:
- https://docs.github.com/en/enterprise-cloud@latest/actions/learn-github-actions/environment-variables#about-environment-variables